src/library/dao/playlistdao.cpp

```diff
@@ -1042,3 +1044,24 @@ void PlaylistDAO::sendToAutoDJ(const QList<TrackId>& trackIds, AutoDJSendLoc loc
             break;
     }
 }
+
+
+QList<QPair<int, int> > PlaylistDAO::getPlaylistTracksPositionRowIds(int playlistID) {
```

Comment:
QPair<int, int>: Which one is the row id and which one is the position? You could easily use the wrong one.